server/scripts/2021TaxSeasonDataScript.php

```diff
@@ -0,0 +1,420 @@
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 19:00:00", 7.5, 6, $siteId);
+		}
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Anderson Library data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertCenterForPeopleInNeedData() {
+	GLOBAL $DB_CONN;
+	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Center for People in Need data has already been inserted');
+	}
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 5; // Manually obtained from PROD DB
+
+		// Tuesdays 11-2
+		$tuesdayDates = getWeeklyDatesFromRange('2021-02-02', '2021-03-23');
+
+		foreach ($tuesdayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 11:00:00", 6, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 6, 6, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 6, 6, $siteId);
+		}
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Center for People in Need data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertLorenEiseleyLibraryData() {
+	GLOBAL $DB_CONN;
+
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Loren Eiseley data has already been inserted');
+	}
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 6; // Manually obtained from PROD DB
+
+		// Thursdays 4-7
+		$thursdayDates = getWeeklyDatesFromRange('2021-02-18', '2021-04-08');
+		foreach ($thursdayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 16:00:00", 6, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 6, 6, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 6, 6, $siteId);
+		}
+
+		// Saturdays 1-4
+		$saturdayDates = getWeeklyDatesFromRange('2021-02-20', '2021-04-03');
+		foreach ($saturdayDates as $date) {
+		
+			$firstAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 6, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 6, 6, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 6, 6, $siteId);
+		}
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Loren Eisely data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertBennettMartinLibraryData() {
+	GLOBAL $DB_CONN;
+
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Bennett Martin data has already been inserted');
+	}
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 7; // Manually obtained from PROD DB
+
+		// Sundays 1-4
+		$sundayDates = getWeeklyDatesFromRange('2021-02-07', '2021-03-14');
+		foreach ($sundayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 6, 6, $siteId);
```

Comment:
Just an aside, since we don't have to insert roles/shifts anymore, we probably don't need to assign the result of `insertAppointmentTime` to a variable anymore. Not a worth a change, just a note